### PR TITLE
Added new email search engine

### PR DESCRIPTION
**Pull request by FarukhS52**
I have gone through the list of email search engines and added a new search engine that was not there.
Hope it helps.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -182,6 +182,7 @@ A curated list of awesome search engines useful during Penetration testing, Vuln
 - [Anymail Finder](https://anymailfinder.com/) - Find verified emails
 - [Tomba.io](https://tomba.io/) - With 430+ million email addresses indexed, effective search filters, and deliverability checks, Tomba's email finder is its most powerful tool
 - [Snov Email Finder](https://snov.io/email-finder) Find any email. Anywhere
+- [EmailSearch](https://emailsearch.io/) - Find any emails and phones from a domain, Linkedin, name, and company. Prospect from 600M business contacts and 20M companies.
```
Comment (edoardottt) [inline README.md:185]: Please change to:
```
- [EmailSearch.io](https://emailsearch.io/) - Find any emails and phones from a domain, Linkedin, name, and company
```

**Review by edoardottt — CHANGES_REQUESTED** [review]
requesting small changes

File: README.md (hunk 1 above)
Comment (FarukhS52) [reply README.md:185]: Okayyy...

**Comment by FarukhS52** [comment]
Hey @edoardottt ,i have made the required changes,kindly review this and let me know if any changes are required

**Comment by FarukhS52** [comment]
Hey @edoardottt ,i have made the required changes,kindly review this and let me know if any changes are required

**Comment by FarukhS52** [comment]
Hey @edoardottt ,i have made the required changes,kindly review this and let me know if any changes are required

**Review by edoardottt — APPROVED** [review]
lgtm

**Pull request merged**